### PR TITLE
Add missing install flags

### PR DESCRIPTION
**Pull request by paigecalvert**
https://deploy-preview-2798--replicated-docs.netlify.app/reference/embedded-cluster-install

### CODE REVIEW

File: docs/reference/embedded-cluster-install.mdx [hunk 1]
```diff
@@ -87,6 +99,13 @@ sudo ./APP_SLUG install --license LICENSE_FILE [flags]
        <ProxyLimitations/>
      </td>
   </tr>
+  <tr>
+     <td>`--private-ca`</td>
+     <td>
+        <p>The path to the trusted certificate authority (CA) certificates used by a Man-In-The-Middle (MITM) proxy that intercepts traffic and issues its own certificates. Using the `--private-ca` flag ensures that the CA is trusted by the installation. KOTS writes the CA bundle provided with the `--private-ca` flag to a ConfigMap in the cluster.</p>
```
Comment (paigecalvert) [inline docs/reference/embedded-cluster-install.mdx:105]: ^ Based on https://app.shortcut.com/replicated/story/111792/support-installing-behind-a-mitm-proxy-that-issues-its-own-certificates, it looks like this flag accepts a CA bundle, then that gets written to a ConfigMap. (I assume KOTS is doing the writing)

**Review by ajp-io — COMMENTED** [review]
made a few minor updates. we've discussed as a team that private-ca isn't just for MITM proxies, so I reworded there a bit. i also asked in Slack if the network interface stuff sounded right, but I'm good with it for now.

**Comment by paigecalvert** [comment]
@ajp-io thanks! I made the update to the private-ca example to include the proxy flags.

**Comment by paigecalvert** [comment]
@ajp-io also saw that Salah confirmed the wording for the network interface flag, so going to merge this

**Pull request merged**